### ISSUE TITLE
Constitution/quasi-constitution protected ground(s) 

### ISSUE THREAD

**Issue by Bazookar** [D3nnis3n/White-Tiger-Feedback#43]
Some grounds of discrimination where if a government entity or a law acts on those differences it would be void.

This would clear these issues and avoid people suing over things that isn’t protected by law. Currently, the only ground that makes sense to me is Citizenships/residency (eg citizen of sondaica gets benefit or extra tax etc). 

If player wishes: inactive, registered company, specialty can be listed. I personally disagree with the 3 grounds I just listed, but that’s just me.

**Comment by D3nnis3n** [comment]
The problem with that is, that there is nothing that is generally and in all cases discriminatory. It always depends on the single case. That's also how I made rulings so far. While Daniel would wish that stuff based on residency would always be unconstitutional, it was never my intention for that to be the case.

**Comment by Bazookar** [comment]
That is one way to go. The suggestion is by no means perfect. The goal of the suggestion was s to make players life’s easier so they don’t have to think about “do I sue on this or nah”. 

If in your view nothing on eco is so black and white, then obviously this proposal isn’t useful.

**Comment by Bazookar** [comment]
The IRL equivalent of the protected grounds would be human rights

**Comment by D3nnis3n** [comment]
Those are already in Basic Rights. And so far, they are not black or white
either, seeing half the world doesnt have such rights or different ones.

**Comment by D3nnis3n** [comment]
Closing this as invalid, as we already got basic rights and potential cases of discriminatory laws are supposed to be rather fluent, as at least I do not wish to lock out specific things by pure fingersnip, there always is a good reason to support some states more than the other, for example when it's poor Virgata. Equity required different treatment, so it's something that needs to be decided case-by-case. I also didn't see too many cases of such disputes lately, more of other things that weren't there before.
